pyannote/audio/interactive/diff/recipe.py

```diff
@@ -0,0 +1,239 @@
+        errors = errors.rename_labels(newLabels)
+        errors = errors.subset(["correct"], invert=True)
+        if listerrors[0] or listerrors[1] or listerrors[2]:
+            if not listerrors[0]:
+                errors = errors.subset(["false alarm"], invert=True)
+            if not listerrors[1]:
+                errors = errors.subset(["confusion"], invert=True)
+            if not listerrors[2]:
+                errors = errors.subset(["missed detection"], invert=True)
+        clean_errors = Annotation()
+        for segment, track, label in errors.itertracks(yield_label=True):
+            if segment.duration * 1000 > minduration:
+                clean_errors[segment, track] = label
+        errors = clean_errors
+        files_errors[filename] = errors
+
+    audio_for_prodigy = AudioForProdigy()
+    audio_for_pipeline = Audio(mono=True)
+    chunks = get_chunks(source, chunk_duration=chunk)
+    chunks = list(chunks)
+    chunks = sorted(
+        chunks,
+        key=lambda k: max(
+            (
+                files_errors[k[0]["text"]]
+                .crop(k[1], mode="intersection")
+                .label_duration(e)
+                for e in files_errors[k[0]["text"]]
+                .crop(k[1], mode="intersection")
+                .labels()
+            ),
+            default=0,
+        ),
+        reverse=True,
+    )
+
+    for file, excerpt in chunks:
+        path = file["path"]
+        filename = file["text"]
+        text = f"{filename} [{excerpt.start:.1f} - {excerpt.end:.1f}]"
+        duration = audio_for_pipeline.get_duration(path)
+        # load audio excerpt
+        waveform, sample_rate = audio_for_pipeline.crop(path, excerpt, mode="pad")
+        # load audio excerpt for visualization in Prodigy
+        audio = audio_for_prodigy.crop(path, excerpt)
+
+        audio_spans = get_audio_spans(
+            files_errors[filename], excerpt, Segment(0, duration)
+        )
+        ref = get_audio_spans(reference[filename], excerpt, Segment(0, duration))
+        hyp = get_audio_spans(hypothesis[filename], excerpt, Segment(0, duration))
+
+        yield {
+            "path": path,
+            "text": text,
+            "audio": audio,
+            "audio_spans": audio_spans,
+            "reference": ref,
+            "hypothesis": hyp,
+            "chunk": {"start": excerpt.start, "end": excerpt.end},
+            "meta": {
+                "file": filename,
+                "start": f"{excerpt.start:.1f}",
+                "end": f"{excerpt.end:.1f}",
+            },
+        }
+
+
+@prodigy.recipe(
+    "pyannote.diff",
+    dataset=("Dataset to save annotations to", "positional", None, str),
+    source=(
+        "Path to directory containing audio files whose annotation is to be checked",
+        "positional",
+        None,
+        str,
+    ),
+    reference=("Path to reference file", "positional", None, str),
+    hypothesis=("Path to hypothesis file ", "positional", None, str),
```

Comment:
Can you clarify that this is expecting RTTM files?
And also a word about what the `uri` field should contain?